config/puma.rb

```diff
@@ -3,18 +3,18 @@
 if ENV['RAILS_ENV'] == 'production'
 
   # First, you need to change these below to your situation.
-  APP_ROOT = '/home/discourse/discourse'
+  app_root = ENV["APP_ROOT"] ||= '/home/discourse/discourse'
```

Comment:
It shouldn't be necessary to change `APP_ROOT` from a constant to a variable for this. It might be because you used `||=` rather than something like:

`APP_ROOT = ENV["APP_ROOT"] || "/home/discourse/discourse"`

Reply:
Thanks for your feedback. I used a style similar to [this](https://github.com/discourse/discourse/blob/main/config/discourse.pill.sample#L2) file.
What's your suggestion? Would you like to see `app_root` changed back to `APP_ROOT`?

Reply:
Yes, I would. And use `||` instead of `||=`

Reply:
Thanks!

Reply:
done